### PR TITLE
[release-4.14] OCPBUGS-33635: restrict Masthead logo to max-height to 60px

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm

**Comment by yanpzhan** [comment]
Checked on the cluster launched against the pr, now a big logo could be restricted in max-height of 60px, the bug is fixed.
/label qe-approved
/label cherry-pick-approved

File: frontend/public/components/masthead.jsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const Masthead = React.memo(({ isMastheadStacked, isNavOpen, onNavToggle
       </MastheadToggle>
       <MastheadMain>
         <MastheadBrand {...logoProps}>
-          <Brand src={details.logoImg} alt={details.productName} />
+          <Brand src={details.logoImg} alt={details.productName} data-test="brand-image" />
```
Comment (rhamilto) [inline frontend/public/components/masthead.jsx:84]: This change was unnecessary since I removed the related Cypress test update in order to pass in CI.

**Pull request merged**